packages/main/test/specs/Calendar.spec.js

```diff
@@ -361,4 +361,56 @@ describe("Calendar general interaction", () => {
 		assert.strictEqual(await yaerInfo[0].getText(), "1416 AH", "First text of year set in the button")
 		assert.strictEqual(await yaerInfo[1].getText(), "1995 - 1996", "Second text of year set in the button")
 	});
+
+	it("Min and max dates are set without format-pattern by using ISO (YYYY-MM-dd) format", async () => {
+		await browser.url("test/pages/Calendar.html");
+
+		const calendar = await browser.$("#calendar6");
+		await calendar.setAttribute("max-date", new Date(Date.UTC(2024, 9, 4, 0, 0, 0)).toISOString().split("T")[0]); // sets the max date to 2024-10-04
+		
+		const yearButton = await calendar.shadow$("ui5-calendar-header").shadow$(`div[data-ui5-cal-header-btn-year]`);
+		await yearButton.click();
+
+		const year2025 = await calendar.shadow$("ui5-yearpicker").shadow$$(`div[role="gridcell"] span`).find(async span => {
+			const text = await span.getText();
+			return text === "2025";
+		}).parentElement();
+
+		assert.strictEqual(await year2025.hasClass("ui5-yp-item--disabled"), true, "Year 2025 is disabled");
+	});
+
+	it("Min and max dates are NOT set without format-pattern, because are not in ISO format (YYYY-MM-dd)", async () => {
+		await browser.url("test/pages/Calendar.html");
+
+		const calendar = await browser.$("#calendar1");
+		const nextButton = await calendar.shadow$("ui5-calendar-header").shadow$("[data-ui5-cal-header-btn-next]");
+		const prevButton = await calendar.shadow$("ui5-calendar-header").shadow$("[data-ui5-cal-header-btn-prev]");
+		const yearButton = await calendar.shadow$("ui5-calendar-header").shadow$(`div[data-ui5-cal-header-btn-year]`);
+		// setting the min and max dates both to a valid format date, but not in the valid ISO format.
+		await calendar.setAttribute("max-date", new Date(Date.UTC(2024, 9, 4, 0, 0, 0)).valueOf() / 1000);
```

Comment:
Why is this devided by 1000?

Reply:
My idea here was to set the `min-date` and `max-date` both to a valid dates ( in the case of `max-date` I divided it by 1000 to provide it as timestamp (which thinking of it now is - wrong approach) ) and to showcase that the min and max dates are not being set if the format is not the `YYYY-MM-dd` one. 

I am going to correct that, thanks !